visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts

```diff
@@ -316,13 +335,15 @@ describe("codeMapMouseEventService", () => {
 				event = { button: ClickType.LeftClick }
 			})
 
-			it("should not do anything when no building is hovered and nothing is selected", () => {
+			it("should not do anything when no building is hovered and nothing is selected, but should call changeCursorIndicator with default", () => {
+				CodeMapMouseEventService.changeCursorIndicator = jest.fn()
 				threeSceneService.getHighlightedBuilding = jest.fn()
 				threeSceneService.getSelectedBuilding = jest.fn()
 
 				codeMapMouseEventService.onDocumentMouseUp(event)
 
 				expect(threeSceneService.selectBuilding).not.toHaveBeenCalled()
+				expect(CodeMapMouseEventService.changeCursorIndicator).toHaveBeenCalledWith(CursorType.Default)
```

Comment:
Instead of checking if the method is called, do something more intuitive and check if  `document.body.style.cursor` has been set correctly. Mocking methods is useful, when they're from another file / service.

Reply:
This applies to the other tests as well.